.linters/cpp/checkKeyword.py

```diff
@@ -0,0 +1,139 @@
+# signout--coding:utf-8--
+
+# Copyright (c) 2019 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+
+import os
+import sys
+import re
+
+
+PASER_FILE_PATH = 'src/parser/parser.yy'
+SCANNER_FILE_PATH = 'src/parser/scanner.lex'
+
+reserved_key_words = [
+    'KW_GO',
+    'KW_AS',
+    'KW_TO',
+    'KW_OR',
+    'KW_AND',
+    'KW_XOR',
+    'KW_USE',
+    'KW_SET',
+    'KW_FROM',
+    'KW_WHERE',
+    'KW_MATCH',
+    'KW_INSERT',
+    'KW_YIELD',
+    'KW_RETURN',
+    'KW_DESCRIBE',
+    'KW_DESC',
+    'KW_VERTEX',
+    'KW_EDGE',
+    'KW_EDGES',
+    'KW_UPDATE',
+    'KW_UPSERT',
+    'KW_WHEN',
+    'KW_DELETE',
+    'KW_FIND',
+    'KW_LOOKUP',
+    'KW_ALTER',
+    'KW_STEPS',
+    'KW_OVER',
+    'KW_UPTO',
+    'KW_REVERSELY',
+    'KW_INDEX',
+    'KW_INDEXES',
+    'KW_REBUILD',
+    'KW_INT',
+    'KW_BIGINT',
+    'KW_DOUBLE',
+    'KW_STRING',
+    'KW_BOOL',
+    'KW_TIMESTAMP',
+    'KW_TAG',
+    'KW_TAGS',
+    'KW_UNION',
+    'KW_INTERSECT',
+    'KW_MINUS',
+    'KW_NO',
+    'KW_OVERWRITE',
+    'KW_SHOW',
+    'KW_ADD',
+    'KW_CREATE',
+    'KW_DROP',
+    'KW_REMOVE',
+    'KW_IF',
+    'KW_NOT',
+    'KW_EXISTS',
+    'KW_WITH',
+    'KW_CHANGE',
+    'KW_GRANT',
+    'KW_REVOKE',
+    'KW_ON',
+    'KW_BY',
+    'KW_IN',
+    'KW_DOWNLOAD',
+    'KW_GET',
+    'KW_OF',
+    'KW_ORDER',
+    'KW_INGEST',
+    'KW_COMPACT',
+    'KW_FLUSH',
+    'KW_SUBMIT',
+    'KW_ASC',
+    'KW_DISTINCT',
+    'KW_FETCH',
+    'KW_PROP',
+    'KW_BALANCE',
+    'KW_STOP',
+    'KW_LIMIT',
+    'KW_OFFSET',
+    'KW_GROUP',
+    'KW_IS',
+    'KW_NULL',
+    'KW_FORCE',
+    'KW_RECOVER'
+]
+
```

Comment:
Good!

I think when defining tokens in `scanner.lex`，  you should distinguish between `keywords `and` unreserved keywords`,    rather than mixing them together.

In this way, when adding a token later,  you know that it is added to the `keyword `or the` unreserved keyword.`